task/python-sdk-azure/0.1/python-sdk-azure.yaml

```diff
@@ -0,0 +1,47 @@
+apiVersion: tekton.dev/v1
+kind: Task
+metadata:
+  name: python-azure-sdk
+  labels:
+    app.kubernetes.io/version: "0.1"
+  annotations:
+    tekton.dev/pipelines.minVersion: "0.54.0"
+    tekton.dev/categories: sdk
```

Comment:
I doubt that the `category: sdk` exists. https://github.com/tektoncd/hub/blob/main/config.yaml#L15 are the available categories.

Reply:
done